Streaming.cpp

```diff
@@ -132,8 +132,9 @@ void SoapyRTLSDR::rx_callback(unsigned char *buf, uint32_t len)
     //increment buffers available under lock
     //to avoid race in acquireReadBuffer wait
     {
-        std::lock_guard<std::mutex> lock(_buf_mutex);
-        _buf_count++;
+    std::lock_guard<std::mutex> lock(_buf_mutex);
+    _buf_count++;
+
     }
 
```

Comment:
Typo fixes look good, but ^^^ is this just whitespace changes from debug prints?